Automattic/jetpack
# Payment Buttons: Add justification, orientation and block-spacing controls

Fixes #25927

#### Changes proposed in this Pull Request:

Add grid layout options to Payment Buttons, making the design just as configurable as Core Buttons. Specifically this brings the following controls:
 * Justification
 * Orientation
 * Block spacing

#### Other information:



#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Create a [JN](https://href.li/?https://jurassic.ninja/create/?jetpack-beta&branch=master) / wpcom site with a paid plan
* Insert some Payment Blocks, go through the stripe connection etc
* Publish the post and open it in a new window
* Use JP beta to switch your site to this branch / apply this patch to your sandbox
* Open the published post in a new window and compare the two - they should look identical
* Open the published post in the editor, it should load as normal - no errors
* Twiddle with the new options: the preview should update accordingly and after saving so should the published post.

Before | After
-------|------
![image](https://user-images.githubusercontent.com/93301/190361773-e600ca4a-610c-4792-9f79-0433c17e3c3d.png) | ![image](https://user-images.githubusercontent.com/93301/190361841-9704e355-cd33-4ca8-86a7-aa245b62feba.png)
